src/Notifier.php

```diff
@@ -23,9 +23,10 @@ class Notifier
     public function __construct($opt = [])
     {
         // TODO: test that projectId and projectKey exists
-        $this->opt = array_merge($opt, [
+        $this->opt = array_merge([
             'host' => 'api.airbrake.io',
-        ]);
+            'secure' => true,
```

Comment:
I added the secure parameter according to original airbraike documentation: https://github.com/airbrake/airbrake/blob/master/lib/airbrake/configuration.rb#L25
I think it is a good idea to keep it as much similar as it is possible